Hearthstone Deck Tracker/Windows/OverlayWindow.Tooltips.cs

```diff
@@ -313,7 +313,7 @@ private void SetFlavorTextEntity(Entity entity)
 		{
 			try
 			{
-				if(!Config.Instance.ShowFlavorText || entity?.Card == null)
+				if(!Config.Instance.ShowFlavorText || entity?.Card?.FormattedFlavorText == "")
```

Comment:
I originally thought the `?` after `card` would return false if card was null, however I agree `IsNullOrEmpty` is probably much cleaner.